interface/modules/custom_modules/oe-module-comlink-telehealth/moduleConfig.php

```diff
@@ -37,11 +38,12 @@
     $content = trim(file_get_contents("php://input"));
     $credentials = json_decode($content, true);
     $cryptoGen = new CryptoGen();
-    $items['comlink_telehealth_registration_uri'] = $credentials['registrationUri'];
-    $items['comlink_telehealth_video_uri'] = $credentials['videoApiUri'];
-    $items['comlink_telehealth_user_id'] = $credentials['ctsiOrgUid'];
-    $items['comlink_telehealth_user_password'] = $cryptoGen->encryptStandard(trim($credentials['ctsiOrgPwd']));
-    $items['comlink_telehealth_cms_id'] = $credentials['ctsiOrgId'];
+    $items[TelehealthGlobalConfig::COMLINK_VIDEO_REGISTRATION_API] = $credentials['registrationUri'];
+    $items[TelehealthGlobalConfig::COMLINK_VIDEO_TELEHEALTH_API] = $credentials['videoApiUri'];
+    $items[TelehealthGlobalConfig::COMLINK_VIDEO_API_USER_ID] = $credentials['ctsiOrgUid'];
+    $items[TelehealthGlobalConfig::COMLINK_VIDEO_API_USER_PASSWORD] = $cryptoGen->encryptStandard(trim($credentials['ctsiOrgPwd']));
+    $items[TelehealthGlobalConfig::COMLINK_VIDEO_TELEHEALTH_CMS_ID] = $credentials['ctsiOrgId'];
+    $items[TelehealthGlobalConfig::COMLINK_TELEHEALTH_PAYMENT_SUBSCRIPTION_ID] = $credentials['paypal_subscription_id'] ?? null;
     // Save to globals table.
```

Comment:
Recommend these default to empty string if undefined so exception won't get  thrown by sequel engine for writing null values to this table.
Or put a null key/value check in foreach.
Sorry I missed that in my commits.

I'm testing this on my test jig for pass through of new array key from server.
Will let ya know

Reply:
Fixed.